lte/gateway/c/session_manager/SessionManagerServer.cpp

```diff
@@ -58,6 +58,17 @@ void LocalSessionManagerAsyncService::init_call_data() {
   new BindPolicy2BearerCallData(cq_.get(), *this, *handler_);
 }
 
+/*Landing object invocation object call for 5G*/
+AmfPduSessionSmContextAsyncService::AmfPduSessionSmContextAsyncService(
+    std::unique_ptr<ServerCompletionQueue> cq,
+    std::unique_ptr<SetMessageManagerHandler> handler)
+    : AsyncService(std::move(cq)), handler_(std::move(handler)) {}
+
+void AmfPduSessionSmContextAsyncService::init_call_data() {
+  new SetAmfSessionContextCallData(cq_.get(), *this, *handler_);
+  MLOG(MINFO) << "ACL5g@74amf sessioncontext calldata";
```

Comment:
maybe `Initializing new call data for SetAmfSessionContext`?

Reply:
yes Marie, it is initializing new call data for SetAmfSessionContext.

Reply:
@rupa455 Sorry, I just meant the log is not very clear, so maybe modify the log so that it explicitly says it is initializing a new call data.

Reply:
okay Marie, i will modify it.